### ISSUE TITLE
Swell, little swell, or closed accent indication for Lines Palette

### ISSUE THREAD

**Issue by slviola** [musescore/MuseScore#16149]
In the Lines Palette, please add a single item that would be a pair of hairpins (cresc-dim) that can be anchored on one note or on several notes, especially ones that are tied together. This indication is very commonly used, especially in 19th century classical music. Normally, for this specific usage the cresc & dim hairpins are symetrical and also vertically aligned, but the overall width and horizontal placement should be adjustable. This is an old, brief discussion: https://musescore.org/en/node/126021
In that discussion was this link, with descriptions that I thought were helpful.
https://www.dolmetsch.com/musicalsymbols.htm "swell, closed hairpins (over a phrase) or closed accent (over a single note): to increase volume and then die away in the duration of a single note or short phrase also called messa di voce (Italian) or mise de voix (French)." or "in nineteenth-century German non-vocal music the < > sign can represent a stress or accent as opposed to a crescendo followed by a decrescendo music. In such a case, the marking apparently indicates a kind of "warm", not too powerful, accent with implication of vibrato where appropriate [suggested by Ron Evans]"

For playback purposes, implementation could be subject to much debate and possibly complicated. If anything like a modest, rounded swell isn't possible, I would suggest something like a 10% increase in volume for the duration of the note as a short term solution. This is the kind of indication that is so open to performer interpreation that making everyone happy about the playback solution isn't like to be possible. Sorry 'bout that in advance... ;-) 
Thanks.

**Comment by oktophonie** [comment]
Nice idea. I'll mention that there is a messa di voce symbol available from the master palette:
![image](https://user-images.githubusercontent.com/20604032/215700261-7c06fac5-c576-4965-b829-97c5a307e402.png)

Not perfect, as it's of a fixed size, but perhaps useful in some situations.

**Comment by slviola** [comment]
Thank-you! That is actually what I've been looking for. (I had to click on More in the Articulation palette to find it, which of course I was sure I had done 5 times. sigh). However, what I'd like now is the ability to significantly control the size. Use of that symbol is quite common on tied notes, including of course ties across bar-lines. The small version currently available looks too small to me even for a single shorter note, and really doesn't make visual sense at all on a longer note or on a tied note. I need a version of the symbol that I can spread out horizontally. It would also be preferable to be able to control it's vertical size. Thanks for working on this.

**Comment by MarcSabatella** [comment]
Meanwhile, this is possible with a workaround - just add invisible rests in another voice, attach regular hairpins to the those, and use an invisible dynamic marking to control the actual volume.  Hopefully that won’t be necessary for *too* much longer, but it does work well.

**Comment by slviola** [comment]
Yes, doing the indication with regular hairpins is possible, but time consuming to get them symetrical and aligned just right. And if the score has many in each part... life is too short. (As far as playback, I know that's important for others, but for me all I really care about at this point is getting them to look just right for each spot reasonably efficiently.) Thanks for working on this.
